packages/devtools_app/lib/src/ui/filter.dart

```diff
@@ -300,7 +298,7 @@ class QueryFilterArgument {
     return false;
   }
 
-  bool matchesValue(String dataValue, {bool substringMatch = false}) {
+  bool matchesValue(String? dataValue, {bool substringMatch = false}) {
```

Comment:
make this non-nullable since we aren't doing any null checking on this value. Deal with the error upstream to ensure we aren't passing a null value for dataValue here.

Reply:
done